controller/instance_manager_controller.go

```diff
@@ -1248,7 +1254,7 @@ func (imc *InstanceManagerController) enqueueSettingChange(obj interface{}) {
 	imc.enqueueInstanceManagersForNode(imc.controllerID)
 }
 
-func (imc *InstanceManagerController) cleanupInstanceManager(imName string) error {
+func (imc *InstanceManagerController) cleanupInstanceManagerPod(imName string) error {
```

Comment:
This function deletes the InstanceManager objects directly, can we just remain the original name?

Reply:
Yes, removed. Thanks.